app/assets/javascripts/admin/templates/components/simple-list.hbs

```diff
@@ -0,0 +1,40 @@
+{{#if collection}}
+  <div class="values">
+    {{#each collection as |value index|}}
+      <div data-index={{index}} class="value">
+        {{d-button
```

Comment:
indentation of both is wrong here

Reply:
Hmm, I was forced to do this by pre-commit hooks though, before I had this indentation, there were linting issues: 

<img width="1131" alt="image" src="https://user-images.githubusercontent.com/368961/83643637-42004e00-a57e-11ea-8843-a48532bdcfcd.png">

Reply:
correct indentation is:

```
{{input
  title=value
  value=value
  class="value-input"
  focus-out=(action "changeValue" index)
}}
```